src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -98,8 +98,11 @@ internal static class WellKnownTypeNames
         public const string SystemAttributeTargets = "System.AttributeTargets";
         public const string SystemAttributeUsageAttribute = "System.AttributeUsageAttribute";
         public const string SystemBoolean = "System.Boolean";
+        public const string SystemBuffer = "System.Buffer";
         public const string SystemBuffersMemoryManager1 = "System.Buffers.MemoryManager`1";
+        public const string SystemArray = "System.Array";
```

Comment:
Did you forget to push changes to this file? `SystemArray` needs to go further to the top I think?